source/fundamentals/crud/read/limit.txt

```diff
@@ -110,6 +110,29 @@ longest books:
 You can combine ``skip()`` and ``limit()`` in this way to implement paging for your
 collection, returning only small subsets of the collection at one time.
 
+.. note::
```

Comment:
@ccho-mongodb this is probably something you should review, since it's the sort of thing we should standardize on across all of our documentation.

Reply:
Sure, talked about this with Katie earlier today. Thanks for bringing it up, I didn't know about this before then. I created a ticket for DevEd to update Node.js documentation and will need to discuss with the driver engineers for the rest of the documentation. I'll have to look into how Server Docs team wants to provide messaging for this as well.